contracts/lib/WhiteList.sol

```diff
@@ -139,4 +139,25 @@ contract WhiteList is
     {
         return addresses;
     }
-}
\ No newline at end of file
+
+    /**
+     * Verifies an array of addresses against the whitelist
+     *
+     * @return address[]      Array of addresses to verify
```

Comment:
should be param and add a return javadoc

Reply:
+1